c/include/ml-api-common.h

```diff
@@ -127,6 +127,7 @@ typedef void *ml_tensors_data_h;
 /**
  * @brief Possible data element types of tensor in NNStreamer.
  * @since_tizen 5.5
+ * @todo add BFLOAT16 when nnstreamer is ready for it.
```

Comment:
can you move this tag (todo) into c file? (exported header file so may raise an issue in acr process)

Reply:
Updated. Thanks.